pingcap/tidb
# server/http: fix scatter table API and add document

### What problem does this PR solve? 
do not need to escape in the post request.

### What is changed and how it works?
remove URL escape.
add document.

### Check List 

Tests 
 - Manual test (add detailed scripts or steps below)
check in `pd-ctl`:
before:
```
  {
      "type": "scatter-range",
      "args": [
        "t%80%00%00%00%00%00%00%FF%1F_r%00%00%00%00%00%FF%00%00%00%00%00%00%00%00%FA",
        "t%80%00%00%00%00%00%00%FF%1F_r%FF%FF%FF%FF%FF%FF%FF%FF%FF%00%00%00%00%00%FA",
        "test3"
      ],
      "disable": false
    },

```
after:
```
 {
      "type": "scatter-range",
      "args": [
        "t�\u0000\u0000\u0000\u0000\u0000\u0000�!_r\u0000\u0000\u0000\u0000\u0000�\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000�",
        "t�\u0000\u0000\u0000\u0000\u0000\u0000�!_r���������\u0000\u0000\u0000\u0000\u0000�",
        "test5"
      ],
      "disable": false
    },
```


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/pingcap/tidb/8424)
